cnd/cnd.lexer/src/org/netbeans/cnd/api/lexer/CndLexerUtilities.java

```diff
@@ -466,13 +470,17 @@ public static Filter<?> getFilter(Language<?> language, /*can be null*/CndLangua
                     case C89:
                     case C99:
                     case C11:
+                    case C17:
+                    case C23:
                         CndUtils.assertTrue(false, "Unexpected flavor " + preferred + " for C++ Language"); // NOI18N
                         break;
                     case CPP98:
                         return CndLexerUtilities.getGccCppFilter();
                     case CPP11:
                     case CPP14:
                     case CPP17:
+                    case CPP20:
+                    case CPP23:
```

Comment:
Similar comment as above, the idea is to be able to track these `// TODO-CND` comments in the future in case we want to improve `CndLexerUtilities` in the future.